### PR TITLE
Helm chart deployment for eth_subscribe server

### LINKED ISSUE #1296: New helm chart deployment for eth_subscribe server
### Problem

Currently the helm chart doesn't support deployment of the websocket server

### Solution

- Add a helm chart for deployment of websocket server

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by isavov**
**Description**:
This PR adds a new helm chart in order to support deployment of the eth_subscribe/websocket interface for the relay.
* Add new helm chart

**Related issue(s)**:

Fixes #1296 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: charts/hedera-json-rpc-relay-websocket/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "json-rpc-relay-websocket.fullname" . }}
+  labels:
+    app:  {{ template "json-rpc-relay-websocket.name" . }}
+    {{ include "json-rpc-relay-websocket.labels" . | nindent 4 }}
+data:
```
Comment (Nana-EC) [inline charts/hedera-json-rpc-relay-websocket/templates/configmap.yaml:8]: A lot of configs below are not utilized in the websocket server package.
Can you reduce this to only the ones utilized by the websocket server as the default configuration

File: charts/hedera-json-rpc-relay-websocket/templates/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (Nana-EC) [inline charts/hedera-json-rpc-relay-websocket/templates/secret.yaml:1]: Q: Is this resource needed?
The websocket server streams from the mirror node only, so the below SDK config secrets should not be needed

File: charts/hedera-json-rpc-relay-websocket/values.yaml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+affinity: {}
+
+autoscaling:
+  enabled: false
+  minReplicas: 2
+  maxReplicas: 8
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+config:
```
Comment (Nana-EC) [inline charts/hedera-json-rpc-relay-websocket/values.yaml:11]: SImilar to configmap, this should only include configs for the webserver socket

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Nice addition.
Please simplify to cover the websocket server only for this chart

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Nice addition.
Please simplify to cover the websocket server only for this chart

**Comment by isavov** [comment]
After digging deeper into the used and unused ENV variables I've limited them to the absolutely necessary.
In the process I've discovered one unnecessary variable in the `hedera-json-rpc-relay` helm chart and removed it in this PR, as the fix was minor and doesn't need a separate PR

@Nana-EC What should the default value for `WS_MULTIPLE_ADDRESSES_ENABLED` be? I've set it to `false` for now, but I'm not sure if that would be best.

**Comment by Nana-EC** [comment]
> After digging deeper into the used and unused ENV variables I've limited them to the absolutely necessary.
> 
> In the process I've discovered one unnecessary variable in the `hedera-json-rpc-relay` helm chart and removed it in this PR, as the fix was minor and doesn't need a separate PR
> 
> 
> 
> @Nana-EC What should the default value for `WS_MULTIPLE_ADDRESSES_ENABLED` be? I've set it to `false` for now, but I'm not sure if that would be best.

False is the right value to start with. We'll build confidence and when ready switch to true.

**Review by Nana-EC — DISMISSED** [review]
LGTM

**Review by AlfredoG87 — CHANGES_REQUESTED** [review]
LGTM,
just 1 request.

Could you include the new helm chart on the bump-version script, similar to how it is the prior one?

https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/scripts/.bump-version.js#L67-L85

**Review by AlfredoG87 — CHANGES_REQUESTED** [review]
LGTM,
just 1 request.

Could you include the new helm chart on the bump-version script, similar to how it is the prior one?

https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/scripts/.bump-version.js#L67-L85

**Review by ebadiere — COMMENTED** [review]
What does this mean?  Will I not be able to run tests against the ws server in the local node?

File: charts/hedera-json-rpc-relay-websocket/ci/default-values.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Default values for json-rpc-relay in CI. Configured to testnet
```
Comment (Nana-EC) [inline charts/hedera-json-rpc-relay-websocket/ci/default-values.yaml:1]: In a follow up PR we should add a test pod under this charts template to verify it on chart install in CI

**Comment by Nana-EC** [comment]
> What does this mean? Will I not be able to run tests against the ws server in the local node?

Shouldn't affect local runs, just the chart being deployed in production flow.

**Review by AlfredoG87 — APPROVED** [review]
LGTM

**Review by ebadiere — DISMISSED** [review]
LG

**Review by AlfredoG87 — APPROVED** [review]
LGTM

**Comment by isavov** [comment]
> LGTM, just 1 request.
> 
> Could you include the new helm chart on the bump-version script, similar to how it is the prior one?
> 
> https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/scripts/.bump-version.js#L67-L85

Thanks for including it in the bump script, I completely missed that 🙌

File: charts/hedera-json-rpc-relay-websocket/ci/default-values.yaml (hunk 4 above)
Comment (Nana-EC) [reply charts/hedera-json-rpc-relay-websocket/ci/default-values.yaml:1]: @isavov making sure you saw this one to add it to any follow up PRs

**Pull request merged**